compiler/testSuite/14-bug/listconstruct3-fv.bal

```diff
@@ -0,0 +1,4 @@
+import ballerina/io;
```

Comment:
This is an unimplemented feature (see #1003) not a bug.

Reply:
fixed